test/fee-vault-fixes/e2e/FeeVault.t.sol

```diff
@@ -2,10 +2,9 @@
 pragma solidity 0.8.15;
 
 import { CommonTest } from "test/CommonTest.t.sol";
-import { Predeploys } from "@eth-optimism-bedrock/contracts/libraries/Predeploys.sol";
-import { Proxy } from "@eth-optimism-bedrock/contracts/universal/Proxy.sol";
-import { FeeVault as FeeVault_Final } from "@eth-optimism-bedrock/contracts/universal/FeeVault.sol";
-import { L1FeeVault as L1FeeVault_Final } from "@eth-optimism-bedrock/contracts/L2/L1FeeVault.sol";
+import { Predeploys } from "@eth-optimism-bedrock/src/libraries/Predeploys.sol";
+import { Proxy } from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import { L1FeeVault as L1FeeVault_Final, FeeVault as FeeVault_Final } from "@eth-optimism-bedrock/src/L2/L1FeeVault.sol";
```

Comment:
Ran into some memory issues for `FeeVault` when compiling (foundry 0.2.0)
```
Error (7407): Type enum FeeVault.WithdrawalNetwork is not implicitly convertible to expected type enum FeeVault.WithdrawalNetwork.
  --> test/fee-vault-fixes/e2e/FeeVault.t.sol:27:29:
   |
27 |         withdrawalNetwork = L1FeeVault_Final(payable(Predeploys.SEQUENCER_FEE_WALLET)).WITHDRAWAL_NETWORK();
   |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

This import change fixed the error (reference: https://github.com/foundry-rs/foundry/issues/6407#issuecomment-1851226955)